sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_runtime_dim.cpp

```diff
@@ -0,0 +1,20 @@
+//==--- joint_matrix_bf16_fill_k_cache_runtime_dim.cpp  - DPC++ joint_matrix--------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: aspect-ext_intel_matrix
+// XFAIL: arch-intel_gpu_pvc
+
+// RUN: %{build} -o %t_runtime_dim_vnni.out -ffp-model=precise -DRUNTIME_DIM -DVNNI
+// RUN: %{run} %t_runtime_dim_vnni.out 256
+
+// RUN: %{build} -o %t_runtime_dim.out -ffp-model=precise -DRUNTIME_DIM
+// RUN: %{run} %t_runtime_dim.out 256
+
```

Comment:
if the goal of the test is to check runtime dimensions, is it necessary to run it with and without VNNI? Can we limit to one run?
```suggestion
```

Reply:
Thanks for the suggestion! I have removed the one without VNNI.